Project.toml

```
@@ -0,0 +1,19 @@
+name = "UnitfulIntegration"
+uuid = "d000b014-1cca-4823-a42a-b615c320b663"
```

```suggestion
uuid = "6043864f-155a-5464-b5be-64e090a46784"
```
This is the UUID that the package currently has (see https://github.com/JuliaRegistries/General/blob/master/U/UnitfulIntegration/Package.toml).